rmf_websocket/src/rmf_websocket/BroadcastServer.cpp

```diff
@@ -39,10 +39,19 @@ class BroadcastServer::Implementation
   Implementation(
     const int port,
     ApiMessageCallback callback,
-    std::optional<ApiMsgType> msg_selection)
-  : _data(std::make_shared<Data>(std::move(callback),
-      std::move(msg_selection)))
+    std::optional<ApiMsgType> msg_selection,
+    const rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr
+      node_logging_interface)
+  : _data(std::make_shared<Data>(std::move(callback), std::move(msg_selection))),
```

Comment:
```suggestion
  : _data(std::make_shared<Data>(std::move(callback),
    std::move(msg_selection))),
```
nit: line longer than 80 characters